e2e/support/helpers/e2e-dashboard-visualizer-helpers.ts

```diff
@@ -142,9 +142,9 @@ export function showDashcardVisualizerModalSettings(index = 0) {
   });
 }
 
-export function saveDashcardVisualizerModal() {
+export function saveDashcardVisualizerModal(isCreation = false) {
```

Comment:
nit: a boolean argument might be ambigious when looking just at the function call:

```js
// what's the difference? i have to look it up
H.saveDashcardVisualizerModal(true)
H.saveDashcardVisualizerModal(false)
```

We could make it a bit more explicit by using a string argument instead:

```js
H.saveDashcardVisualizerModal("create")
H.saveDashcardVisualizerModal("update")

// or

H.saveDashcardVisualizerModal({ mode: "create" })
H.saveDashcardVisualizerModal({ mode: "update" })

```

Reply:
That's right, I'll make the change. Thanks!